pkg/config/config.go

```diff
@@ -49,3 +52,38 @@ func GetDefaultConfigPath() (string, error) {
 
 	return filepath.Join(home, DefaultConfigPath, DefaultConfigFileName), nil
 }
+
+func GetPathToCalculation(defaultPath string, fileName string) (string, error) {
+	user, err := user.Current()
+	if err != nil {
+		return "", fmt.Errorf("couldn't get user's information: %v", err)
+	}
+	home := user.HomeDir
+
+	if defaultPath == "" {
+		return filepath.Join(home, DefaultConfigPath, fileName), nil
+	} else {
+		if _, err := os.Stat(defaultPath); os.IsNotExist(err) {
+			logrus.Info("Desired path to download the calculations does not exist. The calculations are set to be downloaded into: $HOME/.config/ccboc/{id}")
```

Comment:
When logging, the user should know all the information about the problem. The `desired path` is unclear here. We need to give him the exact path. Also, by handling only the not exists error, we eat all the other potential errors. Instead lets check for all errors and output the error in the screen. There is no need to fallback to the default location, since the user didn't ask for that.

Reply:
Changed it to errors.Is(err, os.ErrNotExist). Hopefully that's enough.

Reply:
No I mean we should check all the errors, and output the error to the user. Don't fall back to the default location.

Reply:
Well, thats what I did in the latest commit. When there is an error, then we fail. I dont fall to the def. location anymore.

Reply:
Yes, but you are checking only one error case. We need to fail in any error.